tags.go

```diff
@@ -9,8 +9,8 @@ import (
 
 // DefaultTagSuffix returns a set of default suggested tags
 // based on the commit ref with an attached suffix.
-func DefaultTagSuffix(ref, suffix string) []string {
-	tags := DefaultTags(ref)
+func DefaultTagSuffix(ref, suffix, defaultBranch string) []string {
```

Comment:
lets reverse the ordering since string functions usually put the prefix/ suffix as the last variable in the function.

```diff
- ref, suffix, defaultBranch string
+ ref, defaultBranch, suffix string
```

Reply:
Done. https://github.com/drone-plugins/drone-docker/pull/160/commits/ff05dfd6dcc3f8a4276b607c0e3f7c6c15f624e5